src/main/groovy/io/seqera/wave/service/scan/ScanRequest.groovy

```diff
@@ -78,6 +80,11 @@ class ScanRequest {
      */
     final Instant creationTime
 
+    /**
+     * Scan request platform identity
+     */
+    final PlatformId identity
```

Comment:
This method should be updated as well (and tests) 

[ScanRequest](https://github.com/seqeralabs/wave/blob/88fea7c69c9ff802612a8250ff43c868355d2690/src/main/groovy/io/seqera/wave/service/scan/ScanRequest.groovy#L82-L94)

Reply:
done